packages/tokens/README.md

```diff
@@ -0,0 +1,81 @@
+# @openfun/cunningham-tokens
+
+## Introduction
+
+**What are design tokens ?**
+
+Design tokens are the fundamental variables defining the precise behavior and rendering of ui components.
+
+For example:
+- The primary color of a text element
+- The standard spacing between two elements
+- The border radius of a button
+- ...
+
+## Getting started
+
+In this section we will install the library and generate the file that contains the design tokens of your app in order to 
+make Cunningham's Design System yours!
+
+Install the lib
+
+```
+yarn add @openfun/cunningham-tokens
+```
+
+Create a file named `cunningham.cjs` at the root of your project
+
+```
+module.exports = {
+  theme: {
+    colors: {
+      primary: 'purple'
+    },
+  },
+};
+```
+
+In this configuration file you can overwrite all the default values of the design system. 
+You can find the default values [here](./src/bin/cunningham.dist.js).
+
+Now add this script to your `package.json`
+
+```
+{
+    "scripts": {
+        ...
+        "build-theme": "cunningham"
+    }
+}
+```
+
```

Comment:
I'll just detail here or just above what is the cunningham CLI. Then mention that `cunnigham -h` can detail all available options.